src/openzeppelin/tests/test_account.cairo

```diff
@@ -192,13 +264,14 @@ fn test__assert_only_self_false() {
 #[test]
 #[available_gas(2000000)]
 fn test_validate_transaction() {
-    // todo: requires mocking TxInfo
+    let account = setup_dispatcher();
```

Comment:
this test is redundant with the one above. not sure what i meant to test with this one